server/util.go

```diff
@@ -245,6 +255,14 @@ func TrapSignal(cleanupFunc func()) {
 	}()
 }
 
+// WaitForQuitSignals waits for SIGINT and SIGTERM and returns.
+func WaitForQuitSignals() ErrorCode {
+	sigs := make(chan os.Signal, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+	sig := <-sigs
+	return ErrorCode{Code: int(sig.(syscall.Signal)) + 128}
```

Comment:
:heart:

Reply:
Why adding 128?

Reply:
Because that is the standard way you handle -signal process termination in UNIX (n+128)